vcf/model.py

```diff
@@ -362,7 +360,7 @@ def heterozygosity(self):
         If there are i alleles with frequency p_i, H=1-sum_i(p_i^2)
         """
         allele_freqs = [1-sum(self.aaf)] + self.aaf
-        return 1 - sum(map(lambda x: x**2, allele_freqs))
+        return 1 - sum([x ** 2 for x in allele_freqs])
```

Comment:
The brackets are not needed here.
```suggestion
        return 1 - sum(x ** 2 for x in allele_freqs)
```